foolbox/zoo/model_loader.py

```diff
@@ -39,7 +40,10 @@ def _import_module(path, module_name='foolbox_model'):
 
 class DefaultLoader(ModelLoader):
 
-    def load(self, path, module_name='foolbox_model'):
-        module = ModelLoader._import_module(path, module_name)
-        model = module.create()
+    def load(self, path, **kwargs):
+        module = ModelLoader._import_module(path)
+        if kwargs:  # Empty dictionaries evaluate fo false
+            model = module.create(**kwargs)
+        else:  # For backwards compatibility
```

Comment:
Are you sure this is needed?
Isn’t unpacking an empty dictionary equivalent anyway?

Reply:
You're absolutely right, I misinterpreted how python would handle the unpacking of the dictionary.